
From: Matthew Dobson <colpatch@us.ibm.com>

NODE_BALANCE_RATE is defined all over the place, but used nowhere.  Let's
remove it.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-alpha/topology.h          |    3 ---
 25-akpm/include/asm-generic/topology.h        |    5 -----
 25-akpm/include/asm-i386/topology.h           |    3 ---
 25-akpm/include/asm-ia64/topology.h           |    3 ---
 25-akpm/include/asm-m32r/topology.h           |    5 -----
 25-akpm/include/asm-mips/mach-ip27/topology.h |    3 ---
 25-akpm/include/asm-ppc64/topology.h          |    3 ---
 25-akpm/include/asm-x86_64/topology.h         |    2 --
 8 files changed, 27 deletions(-)

diff -puN include/asm-alpha/topology.h~sched-remove-node_balance_rate-definitions include/asm-alpha/topology.h
--- 25/include/asm-alpha/topology.h~sched-remove-node_balance_rate-definitions	2004-10-05 21:52:25.825348064 -0700
+++ 25-akpm/include/asm-alpha/topology.h	2004-10-05 21:52:25.839345936 -0700
@@ -39,9 +39,6 @@ static inline cpumask_t node_to_cpumask(
 	return node_cpu_mask;
 }
 
-/* Cross-node load balancing interval. */
-# define NODE_BALANCE_RATE 10
-
 #define pcibus_to_cpumask(bus)	(cpu_online_map)
 
 #else /* CONFIG_NUMA */
diff -puN include/asm-generic/topology.h~sched-remove-node_balance_rate-definitions include/asm-generic/topology.h
--- 25/include/asm-generic/topology.h~sched-remove-node_balance_rate-definitions	2004-10-05 21:52:25.826347912 -0700
+++ 25-akpm/include/asm-generic/topology.h	2004-10-05 21:52:25.839345936 -0700
@@ -45,9 +45,4 @@
 #define pcibus_to_cpumask(bus)	(cpu_online_map)
 #endif
 
-/* Cross-node load balancing interval. */
-#ifndef NODE_BALANCE_RATE
-#define NODE_BALANCE_RATE 10
-#endif
-
 #endif /* _ASM_GENERIC_TOPOLOGY_H */
diff -puN include/asm-i386/topology.h~sched-remove-node_balance_rate-definitions include/asm-i386/topology.h
--- 25/include/asm-i386/topology.h~sched-remove-node_balance_rate-definitions	2004-10-05 21:52:25.828347608 -0700
+++ 25-akpm/include/asm-i386/topology.h	2004-10-05 21:52:25.840345784 -0700
@@ -69,9 +69,6 @@ static inline cpumask_t pcibus_to_cpumas
 /* Node-to-Node distance */
 #define node_distance(from, to) ((from) != (to))
 
-/* Cross-node load balancing interval. */
-#define NODE_BALANCE_RATE 100
-
 /* sched_domains SD_NODE_INIT for NUMAQ machines */
 #define SD_NODE_INIT (struct sched_domain) {		\
 	.span			= CPU_MASK_NONE,	\
diff -puN include/asm-ia64/topology.h~sched-remove-node_balance_rate-definitions include/asm-ia64/topology.h
--- 25/include/asm-ia64/topology.h~sched-remove-node_balance_rate-definitions	2004-10-05 21:52:25.830347304 -0700
+++ 25-akpm/include/asm-ia64/topology.h	2004-10-05 21:52:25.840345784 -0700
@@ -40,9 +40,6 @@
  */
 #define node_to_first_cpu(node) (__ffs(node_to_cpumask(node)))
 
-/* Cross-node load balancing interval. */
-#define NODE_BALANCE_RATE 10
-
 void build_cpu_to_node_map(void);
 
 /* sched_domains SD_NODE_INIT for IA64 NUMA machines */
diff -puN include/asm-m32r/topology.h~sched-remove-node_balance_rate-definitions include/asm-m32r/topology.h
--- 25/include/asm-m32r/topology.h~sched-remove-node_balance_rate-definitions	2004-10-05 21:52:25.832347000 -0700
+++ 25-akpm/include/asm-m32r/topology.h	2004-10-05 21:52:25.840345784 -0700
@@ -45,9 +45,4 @@
 #define pcibus_to_cpumask(bus)	(cpu_online_map)
 #endif
 
-/* Cross-node load balancing interval. */
-#ifndef NODE_BALANCE_RATE
-#define NODE_BALANCE_RATE 10
-#endif
-
 #endif /* _ASM_M32R_TOPOLOGY_H */
diff -puN include/asm-mips/mach-ip27/topology.h~sched-remove-node_balance_rate-definitions include/asm-mips/mach-ip27/topology.h
--- 25/include/asm-mips/mach-ip27/topology.h~sched-remove-node_balance_rate-definitions	2004-10-05 21:52:25.833346848 -0700
+++ 25-akpm/include/asm-mips/mach-ip27/topology.h	2004-10-05 21:52:25.841345632 -0700
@@ -12,7 +12,4 @@
 extern int node_distance(nasid_t nasid_a, nasid_t nasid_b);
 #define node_distance(from, to)	node_distance(from, to)
 
-/* Cross-node load balancing interval. */
-#define NODE_BALANCE_RATE	10
-
 #endif /* _ASM_MACH_TOPOLOGY_H */
diff -puN include/asm-ppc64/topology.h~sched-remove-node_balance_rate-definitions include/asm-ppc64/topology.h
--- 25/include/asm-ppc64/topology.h~sched-remove-node_balance_rate-definitions	2004-10-05 21:52:25.834346696 -0700
+++ 25-akpm/include/asm-ppc64/topology.h	2004-10-05 21:52:25.841345632 -0700
@@ -37,9 +37,6 @@ static inline int node_to_first_cpu(int 
 
 #define nr_cpus_node(node)	(nr_cpus_in_node[node])
 
-/* Cross-node load balancing interval. */
-#define NODE_BALANCE_RATE 10
-
 /* sched_domains SD_NODE_INIT for PPC64 machines */
 #define SD_NODE_INIT (struct sched_domain) {		\
 	.span			= CPU_MASK_NONE,	\
diff -puN include/asm-x86_64/topology.h~sched-remove-node_balance_rate-definitions include/asm-x86_64/topology.h
--- 25/include/asm-x86_64/topology.h~sched-remove-node_balance_rate-definitions	2004-10-05 21:52:25.836346392 -0700
+++ 25-akpm/include/asm-x86_64/topology.h	2004-10-05 21:52:25.841345632 -0700
@@ -32,8 +32,6 @@ static inline cpumask_t __pcibus_to_cpum
 /* broken generic file uses #ifndef later on this */
 #define pcibus_to_cpumask(bus) __pcibus_to_cpumask(bus)
 
-#define NODE_BALANCE_RATE 30	/* CHECKME */ 
-
 #ifdef CONFIG_NUMA
 /* sched_domains SD_NODE_INIT for x86_64 machines */
 #define SD_NODE_INIT (struct sched_domain) {		\
_
